website/docs/error-reference.md

```diff
@@ -53,6 +53,13 @@ and should not be reported.
 
 [#1993]: https://github.com/sorbet/sorbet/pull/1993
 
+## 3011
+
+There was a hash with a duplicated keys. If you are trying to make a sig with 
+duplicated parameter names like `sig {params(_: String, _: Integer).void}`, 
+you need to choose better names, for example
+ `sig {params(_1: String, _2: Integer).void}`.
```

Comment:
````suggestion
There was a Hash literal with duplicated keys.

```ruby
{my_key: 1, my_key: 2} # error: `my_key` is duplicated
```

This error can also be caused when trying to write a `sig` for a method with
duplicated parameter names:

```ruby
sig {params(_: String, _: Integer).void} # error: `_` is duplicated
def foo(_, _); end
```

To write a `sig` for this method, rename the method's parameters to have unique
names:

```ruby
sig {params(_1: String, _2: Integer).void} # ok
def foo(_1, _2); end
```
````

Reply:
Reasoning: We try not to use "you" and "better" in docs ("you" is a bit too close to casting blame, like "the problem is your fault", and "better" is debatable—people might think that `_` is actually a better name by certain metrics).

also full-sized code snippets look nicer.